baybe/utils/basic.py

```diff
@@ -88,3 +89,20 @@ def group_duplicate_values(dictionary: dict[_T, _U]) -> dict[_U, list[_T]]:
     for key, value in dictionary.items():
         group.setdefault(value, []).append(key)
     return {k: v for k, v in group.items() if len(v) > 1}
+
+
+def get_init_attributes(
+    first: type, second: type, ignored_keys: tuple[str, ...] = ("self", "kwargs")
```

Comment:
why not "args" too?

Reply:
See comment at the other thread. Happy to adjust, although I am not sure if it would have any influence.